main.go

```diff
@@ -111,8 +116,15 @@ func run(c *cli.Context) error {
 		go func() { startchan <- struct{}{} }()
 		file := c.String("file")
 		configchan := make(chan bool)
+		// XXX Unwatchable Puppet sources should get events from a timer,
+		// with the same interval as the Puppet agent
+		// (hint: puppet config print runinterval)
 		if !c.Bool("no-watch") {
```

Comment:
On Thu, May 26, 2016 at 8:19 AM, Felix Frank <notifications@github.com>
wrote:

> I figured that it makes sense for --no-watch to suppress recompiling /
> querying the master.
>

Except we won't have puppet watches on the catalog...